### DISCUSSION TITLE
Having Trouble Filtering based off User field

### DISCUSSION THREAD

**Discussion by Warpzal** [pocketbase/pocketbase#508] [Q&A] [answered]
I'm struggling with filtering through a collection. I'm trying to get all records in that collection that have a certain uploader.  I cant tell what I'm doing wrong. Here are a few screenshots of the issue, I'm having. I'm newbie to development so go easy on me.
![Screenshot from 2022-09-16 20-27-39](https://user-images.githubusercontent.com/101024113/190838891-f8e8e93b-b50c-486a-a362-5b1a1daa2f56.png)
![Screenshot from 2022-09-16 20-28-27](https://user-images.githubusercontent.com/101024113/190838892-97ceb0eb-73d2-4ea4-878e-4ff1d4e18ee0.png)

**Comment by ganigeorgiev** [comment, marked as answer]
You have to wrap the `${user.value.userId}` in single or double quotes, otherwise the filter will try to look for a collection field with this name and since it doesn't exist it throws a 400 error for invalid filter.

```
uploader = "${user.value.userId}"
```

**Reply by Warpzal** [reply]
Thanks that did it!
